SalesforceCommerceCloud/pwa-kit
# Add a cache-control header to the PDP

The PDP is not cached, leading to slower load times.

This solution sets the cache control on the PDP to be the same as the PLP.

GUS: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000C5KqbYAF/view

# Description

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

* Adds a cache control header on the PDP

# How to Test-Drive This PR

* Visit the deployed PDP at https://scaffold-pwa-test.mobify-storefront.com/en-GB/product/25686544M
* Verify there is a cache-control header in the response
* Reload the page a few times and observe that the response's x-cache header shows a cache hit


To test performance, we can compare the results with the control environment at https://pwa-kit.mobify-storefront.com/en-GB/product/25686544M

`ab -n 1000 -c 100 https://scaffold-pwa-test.mobify-storefront.com/en-GB/product/25686544M`
`ab -n 1000 -c 100 https://pwa-kit.mobify-storefront.com/en-GB/product/25686544M`

We can also compare the results with other pages on the control environment that are cached, such as the home page or PLP.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
